crates/zed/src/main.rs

```diff
@@ -1003,31 +1001,26 @@ fn watch_themes(fs: Arc<dyn fs::Fs>, cx: &mut AppContext) {
 }
 
 #[cfg(debug_assertions)]
-async fn watch_languages(fs: Arc<dyn fs::Fs>, languages: Arc<LanguageRegistry>) {
+fn watch_file_types(fs: Arc<dyn fs::Fs>, cx: &mut AppContext) {
     use std::time::Duration;
 
-    let reload_debounce = Duration::from_millis(250);
-
-    let mut events = fs
-        .watch("crates/zed/src/languages".as_ref(), reload_debounce)
-        .await;
-
-    while (events.next().await).is_some() {
-        languages.reload();
-    }
-}
+    #[cfg(not(target_os = "windows"))]
+    let path = Path::new("assets/icons/file_icons/file_types.json");
```

Comment:
Lets just make this code always canonicalize, and remove the conditional compilation.

Reply:
Done.